web/packages/teleport/src/Discover/Kubernetes/EnrollEKSCluster/EnrollEksCluster.tsx

```diff
@@ -293,6 +298,17 @@ export function EnrollEksCluster(props: AgentStepProps) {
       {fetchClustersAttempt.status === 'failed' && !hasIamPermError && (
         <Danger mt={3}>{fetchClustersAttempt.statusText}</Danger>
       )}
+      <Text mt={4} mb={-3}>
```

Comment:
What is the purpose of the negative value here? are we doing it to try and not affect the positioning of something else?

Reply:
I copied it from RDS discover where we use similar styled note, I think it's just to make margin to look better

Negative margin:
![image](https://github.com/gravitational/teleport/assets/964862/ebeb3841-ab6d-4434-aff1-09a74323f376)

vs 0 margin

![image](https://github.com/gravitational/teleport/assets/964862/58ebba21-0673-4f86-850b-480165577667)

Not really sure which one looks better, I can remove negative margin if you think it's not needed.

Reply:
its a nit at this point so feel free to ignore, but it'd probably make more sense to reduce the `mt` of the element right below it (the `Select the AWS Region...` Text element) than to use negative margin here.

Reply:
Makes sense, changed the margins 👍